docs/_milestone_.md

```diff
@@ -0,0 +1,85 @@
+# December Milestone
+
+## Overview
+
+- Complete the epics:
+  - `Add Support for Partitioned Database`
+  - `Inclusion of related models`
+  - `Production deployment`
+
+## Goals
+
+- [ ] Inclusion of related models [MVP]
+      https://github.com/strongloop/loopback-next/issues/1352
+
+  - [ ] [5]Reject create/update requests when data contains navigational
+        properties https://github.com/strongloop/loopback-next/issues/3439
```

Comment:
@dhmlau hopefully story https://github.com/strongloop/loopback-next/issues/3439 will be completed in this month. It's a committed story and Agnes is actively working on it, I can take it over if we need one more week(still within Nov.).

Reply:
Thanks. The milestone plan here is tentative, will depend on what's left from Nov.